src/Client/LiveStripeClient.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace MatchBot\Client;
+
+use MatchBot\Domain\Currency;
+use MatchBot\Domain\Money;
+use MatchBot\Domain\StripeCustomerId;
+use Stripe\PaymentIntent;
+use Stripe\StripeClient;
+
+/**
+ * Connects to a real Stripe service, either in test mode or production mode.
+ *
+ */
+class LiveStripeClient implements Stripe
+{
+    public function __construct(private StripeClient $stripeClient)
+    {
+    }
+
+    public function cancelPaymentIntent(string $paymentIntentId): void
+    {
+        $this->stripeClient->paymentIntents->cancel($paymentIntentId);
+    }
+
+    public function updatePaymentIntent(string $paymentIntentId, array $updateData): void
+    {
+        $this->stripeClient->paymentIntents->update($paymentIntentId, $updateData);
```

Comment:
I'm finding it a little counterintuitive that we're not returning the Payment Intent here even though the underlying lib does and we've found that useful in Identity. But as nothing uses it right now, I guess we still slightly benefit from simplifying and not having to think about the return type for the moment.

Reply:
Happy to add the return in if you think it's worth it. I guess the advantage of returning void is it's easier to mock. If we type this as returning PaymentIntent then any test double will also have to return a payment intent. Probably that can be as simple as `willReturn($this->createStub(PaymentIntent::class));` but we might think we have to spend time making a more realistic PI object even though it's not used.

Reply:
Right – probably makes sense to leave it now until we have a real use case.